test/functional/api/window_spec.lua

```diff
@@ -1292,6 +1723,154 @@ describe('API/win', function()
       eq(title, cfg.title)
       eq(footer, cfg.footer)
     end)
+
+    it('includes split for normal windows', function()
+      local win = api.nvim_open_win(0, true, {
+        vertical = true,
+        win = -1,
+      })
+      eq(api.nvim_win_get_config(win).split, 'left')
+      api.nvim_win_set_config(win, {
+        vertical = false,
+        win = -1,
+      })
+      eq(api.nvim_win_get_config(win).split, 'above')
+      api.nvim_win_set_config(win, {
+        split = 'below',
+        win = -1,
+      })
+      eq(api.nvim_win_get_config(win).split, 'below')
+    end)
+
+    it('includes split when splitting with ex commands', function()
+      local win = api.nvim_get_current_win()
+      eq(api.nvim_win_get_config(win).split, 'left')
+
+      command('vsplit')
+      local win2 = api.nvim_get_current_win()
+
+      -- initial window now be marked as right split
+      -- since it was split with a vertical split
+      -- and 'splitright' is false by default
+      eq(api.nvim_win_get_config(win).split, 'right')
+      eq(api.nvim_win_get_config(win2).split, 'left')
+
+      api.nvim_set_option_value('splitbelow', true, {
+        scope = 'global',
+      })
+      api.nvim_win_close(win, true)
+      command('split')
+      local win3 = api.nvim_get_current_win()
+      eq(api.nvim_win_get_config(win3).split, 'below')
```

Comment:
```suggestion
      eq('below', api.nvim_win_get_config(win3).split)
```

Reply:
Thanks, yeah I figured out half way through writing the tests that I was putting the expected value on the wrong side lol, my bad. Thanks for pointing all of these out!